pkg/agent/nodeportlocal/k8s/annotations.go

```diff
@@ -84,6 +85,14 @@ func assignPodAnnotation(pod *corev1.Pod, nodeIP string, containerPort, nodePort
 	pod.Annotations = current
 }
 
+func removePodAnnotation(pod *corev1.Pod) {
+	current := pod.Annotations
+
+	klog.V(2).Infof("Removing annotation from pod: %v", pod.Name)
```

Comment:
all over the project, we capitalize K8s terms (Pod, Node, Service, ...) in comments / log messages / documentations. Please do the same for consistency.

Reply:
Done, thanks